src/webextension/list/manage/components/app.css

```diff
@@ -52,13 +52,6 @@ body > main,
   overflow: auto;
 }
 
-.filter {
-  border: 1px solid #d1d1d2;
-  border-inline-start: 0px;
-  border-radius: 0;
-}
-
-.filter:focus-within,
-.filter:focus-within:hover {
-  box-shadow: none;
+.filter-bar {
+  padding: 1.5ch;
```

Comment:
I'll leave this up to @changecourse, but maybe we should use pixels here? I don't have a strong opinion, but pixels seems like it'd be easier to match the visual designs.